src/py/mat3ra/made/tools/build/interface/builders.py

```diff
@@ -62,7 +62,8 @@ def _update_material_name(self, material: Material, configuration: InterfaceConf
 #                           Simple Interface Builder                                   #
 ########################################################################################
 class SimpleInterfaceBuilderParameters(InterfaceBuilderParameters):
-    scale_film: bool = True
+    scale_film: bool = True  # Whether to scale the film to match the substrate
```

Comment:
Let's put comments above the line where var is defined

Reply:
It's not how we do elsewhere, I think